huxley/core/models.py

```diff
@@ -361,6 +361,20 @@ def remove(assignment_data):
 
         return failed_assignments
 
+    @classmethod
+    def update_assignment(cls, **kwargs):
+        assignment = kwargs['instance']
+        # Check that this assignment already exists
+        if assignment.id:
+            old_assignment = cls.objects.get(id=assignment.id)
+            # Check if school changed
+            if not assignment.school.id == old_assignment.school.id:
+                assignment.rejected = False
+                delegates = Delegate.objects.filter(assignment_id=old_assignment.id)
```

Comment:
The Delegate's overridden save method isn't called, but I don't think it matters because that's ensuring that the delegate's school field and the delegate's assignment's school field are the same. However if the delegate doesn't have an assignment anymore then there's no point in checking that.